src/subapps/admin/components/Projects/ResourceInfoPanel.tsx

```diff
@@ -30,11 +37,24 @@ const ResourceInfoPanel: React.FC<{
           {'-->'}{' '}
           <a href={destination} target="_blank">
             {labelOf(destination)}
-          </a>{' '}
+          </a>
+          {`: ${relation._count}`}
         </li>
       );
-    });
+    };
+    const renderSource = sourcesRelations.map(displayRelations);
+    const renderDestination = destinationRelations.map(displayRelations);
+
+    return (
+      <div>
+        <h2>Out going</h2>
+        {renderSource}
+        <h2>Incoming</h2>
+        {renderDestination}
```

Comment:
I would recommend flipping the order, or at least the arrows to preserve the link direction (form the graph).

Reply:
Are you suggesting that we do <--- for incoming and --> for outgoing?

Reply:
ideally I would prefer swapping the order as it's visually more accurate

Reply:
Based on the order in which the properties appear, I think changing the arrow direction is better and I have updated code to reflect that.